app/views/site/search_gems.html.erb

```diff
@@ -0,0 +1,10 @@
+<h1>Results</h1>
+<%= render :partial => 'search_gems' %>
+
+<ul>
+<% @gems.each do |gem| %>
+  <li>
+    <h1 ><%= gem.gem %></h1>
```

Comment:
I think this should be a link to the view of the gem instead of plain text @linatodi